Game/ui/consolewidget.h

```diff
@@ -27,11 +27,11 @@ class ConsoleWidget : public Tempest::Widget {
 
     Gothic&                   gothic;
 
-    Tempest::UiOverlay*       overlay    = nullptr;
-    const Tempest::Texture2d* background = nullptr;
+    Tempest::UiOverlay*       overlay;
```

Comment:
Why remove inline initialization?

Reply:
Just because I am not a friend of inline stuff in the .h files, the .h file in my opinion only describes the interface.
And all the other variables are already initialized in the constructor of the object, why not those two?